spec/features/following_spec.rb

```diff
@@ -80,7 +80,7 @@
         visit member_follows_path(member)
         expect(page).not_to have_content "#{other_member.login_name}"
         visit member_followers_path(other_member)
-        expect(page).not_to have_content "#{member.login_name}"
+        # expect(page).not_to have_content "#{member.login_name}"
```

Comment:
[Committing commented-out code is generally a bad idea](http://programmers.stackexchange.com/questions/45378/is-commented-out-code-really-always-bad) - if it doesn't need to be there then just delete it, because if we need it later we can get it back from the Git history. In this case, the code shouldn't be removed, because your code changes shouldn't have broken this test (but it appears they have, because the Git conflict markers mean your HAML code is no longer syntactically correct).